README.md

```diff
@@ -1068,6 +1070,42 @@ order to manually set the time unit, you can specify it manually:
 BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 ```
 
```

Comment:
maybe this should be a separate doc under `docs` that is linked to from here. a README over 1k lines is a bit heavy.

Reply:
done.